server/routes/contacts/create/enter-name/createContactEnterNameController.ts

```diff
@@ -6,10 +6,37 @@ import { CreateContactEnterNameSchemaType } from './createContactEnterNameSchema
 export default class EnterNameController implements PageHandler {
   public PAGE_NAME = Page.CREATE_CONTACT_NAME_PAGE
 
+  private TITLES = [
```

Comment:
I've moved this here temporarily as it's very clunky to set the selected value within the nunjucks. To be replaced by reference data soon hopefully